python/test/t_Wilks_std.py

```diff
@@ -5,20 +5,11 @@
 
 TESTPREAMBLE()
 
-try:
-    alpha = Point(2)
-    alpha[0] = 0.90
-    alpha[1] = 0.95
-    beta = Point(2)
-    beta[0] = 0.90
-    beta[1] = 0.95
-    for a in range(2):
-        for b in range(2):
-            print("alpha=", alpha[a], "beta=", beta[b])
-            for i in range(5):
-                print("n(", alpha[a], ",", beta[b], ",", i, ")=",
-                      Wilks.ComputeSampleSize(alpha[a], beta[b], i))
-
-except:
-    import sys
-    print("t_Wilks_std.py", sys.exc_info()[0], sys.exc_info()[1])
+alpha = [0.05 * i for i in range(1, 20)]
+beta = [0.05 * i for i in range(1, 20)]
+for a in alpha:
+    for b in beta:
+        print("alpha=%.2f" % a, "beta=%.2f" % b)
+        for i in range(5):
+            print("n(%.2f" % a, ",%.2f" %b, ",", i, ")=",
+                  Wilks.ComputeSampleSize(a, b, i))
```

Comment:
seems it fails on win32

Reply:
They were there for debugging purpose. They helped me during the development, maybe they would ease to catch future bugs ;-)!

Reply:
I mean the code of the test is good but they fail to pass on win32